### DISCUSSION TITLE
모바일 기종에 따른 월드 오류

### DISCUSSION THREAD

**Discussion by IlsYoungOG** [naverz/zepeto-studio-kor#1238] [5. Q&A]
안녕하세요. 
현재 모바일 기종에 따라 월드에서 오류가 발생하여 문의 남깁니다.
<img width="488" alt="스크린샷 2022-08-29 오후 4 47 47" src="https://user-images.githubusercontent.com/107122738/187151445-bffb6a68-62a4-4620-bb30-429fc487decc.png">

현재 저희 월드에서는 차량 탑승 오류와 조작버튼의 오류인데요,
아이러니하게도 고사양 모바일에서는 차량 탑승을 하지 못할 때가 종종 발생합니다.
뿐만 아니라 조작버튼이 범위를 벗어나는 
문제도 발생합니다.

![IMG_1583](https://user-images.githubusercontent.com/107122738/187151537-1e838ff7-e214-4d0a-b56c-afad92133c75.PNG)

![IMG_1578](https://user-images.githubusercontent.com/107122738/187151739-3b65e9a0-2828-4a8b-8cdd-4fe3965cbe18.PNG)

그러나 저사양 모바일에서는 이와같은 문제가 발생하지 않으며
오류없이 정상작동이 됩니다.

이와같은 오류 혹은 버그가 발생하는 이유는
아직 제페토 Unity 월드의 서버가 안정화되지 않아서 일까요? 이와같은 문제에 대한 해결책을 주신다면 감사하겠습니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 
차량 탑승 관련 스크립트나 재현이 가능한 파일이 있으면 파악에 도움이 될 것 같습니다.

**Reply by IlsYoungOG** [reply]
파일이나 스크립트는 어떠한 형태로 어떻게 전달하면 되는걸까요?

**Reply by zlmrzlo** [reply]
스크립트를 여기에 올리면 친절하게 관련 내용 알려주시더라구요.
아니면 이메일로 따로 보내도 되는 것 같아요.

**Reply by Zepeto-TechEvangelist** [reply]
안녕하세요, 네 이곳에 직접 올려주셔도 되고 공개적으로 올리기가 걱정되신다면 이메일로 보내주셔도 됩니다.
[dl_zepeto_evangelist@naverz-corp.com](mailto:dl_zepeto_evangelist@naverz-corp.com)
말씀하신 증상의 경우 저희쪽에서 재현을 해봐야 파악이 가능한 부분인 것 같습니다.

**Reply by Zepeto-TechEvangelist** [reply]
우선은 해상도별로 화면크기를 각각 잡아주시면 버튼이 안 눌리는 문제는 해결될 것으로 보여집니다.
